params.go

```diff
@@ -509,7 +509,7 @@ func BuildQueryString(opts interface{}) (*url.URL, error) {
 							var s []string
 							for _, k := range v.MapKeys() {
 								value := v.MapIndex(k).String()
-								s = append(s, fmt.Sprintf("'%s':'%s'", k.String(), value))
+								s = append(s, fmt.Sprintf("\"%s\":\"%s\"", k.String(), value))
```

Comment:
I'm not sure, but \\"%s\\" may be replaced by %q

Reply:
yep, you are right! updated my PR to use it :)